CSS_Tasks/Registration_Form/formStyle.css

```diff
@@ -0,0 +1,45 @@
+body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.registration {
+    width: 20%;
+    border-style: groove;
+    background-color: #f5f5f0;
+}
+
+.input-container {
+    margin: 10px 20px 10px 20px;
+}
+
+div.input-container input {
+    width: 100%;
+    height: 25px;
+}
+
+form.registration legend {
+    height: 25px;
+    padding: 10px;
+    margin: -2px -2px 25px -2px;
+    text-transform: uppercase;
+    text-align: center;
+    font-family: Cambria;
+    font-weight: bold;
+    font-size: 120%;
```

Comment:
You may want to use another metric unit for font size than %: https://developer.mozilla.org/en-US/docs/Web/CSS/font-size